### PR TITLE
Added Codealike 

### PR DESCRIPTION
**Pull request by BenjaminAbt**
The Codealike Team sent an email with their benefits for MVPs

### CODE REVIEW

**Comment by xavierdecoster** [comment]
Thanks @BenjaminAbt !

File: _data/perks.yml [hunk 1]
```diff
@@ -175,5 +175,5 @@
 
 - company: "Codealike"
   uri: "https://codealike.com"
-  description: "Sign up to Codealike. Then e-mail or twit to us just saying what your Codealike username and MVP profile page is. Codealike Premium will be free to MVPs forever."
+  description: "Sign up to Codealike. Then e-mail or send us a tweet just saying what your Codealike username and MVP profile page is. Codealike Premium will be free to MVPs forever."
```
Comment (BenjaminAbt) [inline _data/perks.yml]: "twit us" was the original wording in Codealike's email ;-)

Comment (xavierdecoster) [reply _data/perks.yml]: Lol. Want me to change it back?

Comment (BenjaminAbt) [reply _data/perks.yml]: No it was just a comment ;-)

**Pull request merged**